apps/platform/src/components/ui/line-tab.tsx

```diff
@@ -69,6 +75,22 @@ function LineTab({ customID, tabs }: LineTabsProps): React.JSX.Element {
   // const [selected, setSelected] = useState(tabs[0])
   const searchParams = useSearchParams()
   const search = searchParams.get('tab')
+  const pathname = usePathname()
+
+  const getIcon = (tab: string): React.ReactNode => {
```

Comment:
I would recommend you against this. I think there's a string array with the names of the tabs. If it is somehow possible to make that an object array (of tab name and icon), that would look better.

The tab shouldn't know how it would be used if im making any sense.

Reply:
Naa this totally fine

Reply:
cool